public/js/discover/discover.js

```diff
@@ -23,6 +23,30 @@ function setPageEventListeners() {
   );
 }
 
+/**
+ * Sets the search dropdown options to the categories from the database.
+ */
+function setSearchOptions() {
+  const select = document.getElementById('search-dropdown');
+  fetch('/data/categories')
+    .then((data) => data.json())
+    .then((data) => {
+      for (const category of data) {
+        const option = document.createElement('option');
+
+        const parsedCategory = category.replace(/[^a-zA-Z0-9]/g, ' ');
+        const capitalizedCategory =
+          parsedCategory.charAt(0).toUpperCase() + parsedCategory.slice(1);
```

Comment:
nit: Could make this its own separate function as it may be useful later to transform the Category into a human readable format